troubleshooting/faq.md

```diff
@@ -161,6 +162,47 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## What should I do with git dependencies on closed networks?
+
+If you are installing a package that has a git URL as a dependency, npm will fetch that package from the remote git repository itself. npm supports both [git and HTTP URL formats](https://docs.npmjs.com/files/package.json#dependencies), so we will need to work around this in our private registry.
+
+```
+"dependencies" : {
+  "name1" : "git://github.com/user/project.git#commit-ish",
+  "name2" : "git://github.com/user/project.git#commit-ish"
+}
+```
+When you are installing the package in the closed network where a dependency references a git url rather than a package name, you will be unable to access due to no internet connectivity.
```

Comment:
Minor change: "you will be unable to install the dependency since it references an external network address." instead of "you will be unable to access due to no internet connectivity."